coredb-operator/src/controller.rs

```diff
@@ -192,6 +229,21 @@ impl CoreDB {
     }
 }
 
+pub fn is_pod_ready() -> impl Condition<Pod> + 'static {
```

Comment:
Moved this here from the test file because we're making use of it in the main controller loop.

Reply:
perfect